documentation/asciidoc/computers/configuration/hdmi-config.adoc

```diff
@@ -1,6 +1,6 @@
 == HDMI Configuration
 
-In the vast majority of cases, simply plugging your HDMI-equipped monitor into the Raspberry Pi using a standard HDMI cable will automatically lead to the Pi using the best resolution the monitor supports. The Raspberry Pi Zero uses a mini HDMI port, so you will need a mini-HDMI-to-full-size-HDMI lead or adapter. On the Raspberry Pi 4 there are two micro HDMI ports, so you will need either one or two micro-HDMI-to-full-size-HDMI leads or adapters, depending on how many displays you wish to attach. You should connect any HDMI leads before turning on the Raspberry Pi.
+In the vast majority of cases, simply plugging your HDMI-equipped monitor into the Raspberry Pi using a standard HDMI cable will automatically lead to the Pi using the best resolution the monitor supports. The Raspberry Pi Zero, Zero W and Zero 2 W uses a mini HDMI port, so you will need a mini-HDMI-to-full-size-HDMI lead or adapter. On the Raspberry Pi 4 and Raspberry Pi 400 there are two micro HDMI ports, so you will need either one or two micro-HDMI-to-full-size-HDMI leads or adapters, depending on how many displays you wish to attach. You should connect any HDMI leads before turning on the Raspberry Pi.
```

Comment:
When it applies to multiple things, "uses" changes to "use".
So "The Raspberry Pi Zero uses a ..." and "The Raspberry Pi Zero, Zero W and Zero 2 W use a ..."

And perhaps "so you will need either one or two micro-HDMI-to-full-size-HDMI leads or adapters, depending on how many displays you wish to attach." would be more concise as "so you will need a micro-HDMI-to-full-size-HDMI lead or adapter for each display you wish to attach." ?

And perhaps change "automatically lead to" to "automatically result in", just to avoid any ambiguity with all the mention of HDMI leads? :wink: And obviously "the Pi" should be "the Raspberry Pi".

Reply:
@lurch 
Thanks , I will change it.